src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -103,11 +113,15 @@ const ChartsHolder = (props: {
                   </Disclaimer>
                 </>
               )}
+              {!isMobile && getChartSummarys(projection)[1] && (
+                <ShareButtons chartAbove={getChartSummarys(projection)[0]} />
```

Comment:
Hrm...  This `chartAbove` thing strikes me as unfortunate (ideally the component wouldn't need to know this much about its surroundings).  Can you explain why this is necessary, exactly?  Would it be possible / make sense to handle whatever the "has chart above" situation is here (in `ChartsHolder`) so the component doesn't need to deal with it?

Reply:
chartAbove was for styling purposes (different top margin if there was a chart above). I refactored and wrapped the buttons with the header which removed the need for chartAbove!

Reply:
Nice!